infrastructure/lambda/template.yaml

```diff
@@ -854,10 +854,10 @@ Resources:
       ComparisonOperator: GreaterThanThreshold
       TreatMissingData: notBreaching
 
-  NoMatchingEncryptionKeyAlarm:
+  SessionLambdaNoMatchingEncryptionKeyAlarm:
     Type: AWS::CloudWatch::Alarm
     Properties:
-      AlarmName: !Sub "${AWS::StackName}-${Environment}-NoMatchingEncryptionKeyAlarm"
+      AlarmName: !Sub "${AWS::StackName}-${Environment}-SessionLambdaNoMatchingEncryptionKeyAlarm"
```

Comment:
nit' but we haven't been setting names for alarms explicitly and I don't think we need to do so now?

Reply:
Ignore - we have for the jwt stuff